plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointWriter.java

```diff
@@ -279,7 +351,7 @@ private void writeObjectMapAsFields(BlockBuilder blockBuilder, RowType type, int
         blockBuilder.closeEntry();
     }
 
-    private Optional<Map<String, Object>> preprocessMinMaxValues(RowType valuesType, Optional<Map<String, Object>> valuesOptional)
+    private Optional<Map<String, Object>> preprocessMinMaxValues(RowType valuesType, Optional<Map<String, Object>> valuesOptional, boolean jsonToParquet)
```

Comment:
I find the `jsonToParquet`'s flag name rather misleading.

What about using `isJson` ?